cluster/src/main/java/com/hartwig/pipeline/calling/structural/StructuralCaller.java

```diff
@@ -99,68 +99,66 @@ public StructuralCallerOutput run(final SomaticRunMetadata metadata, final Align
                 format("%s/%s.gridss.working", VmDirectories.OUTPUT, basename(referenceBam.getLocalTargetPath()));
         String tumorWorkingDir = format("%s/%s.gridss.working", VmDirectories.OUTPUT, basename(tumorBam.getLocalTargetPath()));
 
-        bash.addCommand(new MkDirCommand(referenceWorkingDir));
-        bash.addCommand(new MkDirCommand(tumorWorkingDir));
-
-        String preprocessSvOutputReferenceBam =
-                format("%s/%s.sv.bam", referenceWorkingDir, basename(referenceBam.getLocalTargetPath()));
-        String preprocessSvOutputTumorBam = format("%s/%s.sv.bam", tumorWorkingDir, basename(tumorBam.getLocalTargetPath()));
-
         String configurationFile = gridssConfigFiles.find("properties");
         String blacklist = gridssConfigFiles.find("bed");
 
         String refBamPath = referenceBam.getLocalTargetPath();
         String tumorBamPath = tumorBam.getLocalTargetPath();
-        new Preprocess(refBamPath, referenceWorkingDir, referenceSampleName, referenceGenomePath, preprocessSvOutputReferenceBam)
+        new Preprocess(refBamPath, referenceWorkingDir, referenceSampleName, referenceGenomePath)
```

Comment:
Still think we should be doing something with the return value of these two

Reply:
Yeah I agree. Hence my interest in running an A/B against the `master` version. I think something must have got missed somewhere. Suspect there may be more to come here.

Reply:
Ok!